### DISCUSSION TITLE
Transferring nix-bundle to nix-community

### DISCUSSION THREAD

**Discussion by Artturin** [nix-community/infra#1506] [General]
@matthewbauer Would you be willing to transfer `nix-bundle` to `nix-community`?

The repo has had issues and PRs piled up for years and every time I try bundlers I find issues that are already fixed in PRs, so I would like to be a co-maintainer.

**Comment by matthewbauer** [comment]
Yeah that's fine with me! I can do it right now.

**Reply by Artturin** [reply]
Awesome, thank you. Can you also add me as a `maintain` collaborator?

**Reply by Artturin** [reply]
Ah and @tomberek too since he's the maintainer of https://github.com/NixOS/bundlers (and I'm now a co-maintainer)

**Reply by matthewbauer** [reply]
On GitHub settings? I think I may not be able to do that since it is now in nix-community. One of the GitHub owners may have to do it.

**Reply by Artturin** [reply]
I don't remember if anyone gave me admin `nixos-gen-config` but I can manage it, I however did create it in nix-community directly instead of transferring it IIRC.
![image](https://github.com/user-attachments/assets/5d368ed9-8934-4b10-88e9-2548ce0bdafa)

**Reply by zowoq** [reply]
@matthewbauer I've given you admin on the repo.

**Reply by matthewbauer** [reply]
Thanks zowoq

**Reply by matthewbauer** [reply]
Ok I added you, artturin
